packages/ses/console-tools.js

```diff
@@ -0,0 +1,21 @@
+import { makeCausalConsoleFromLoggerMaker } from './src/error/console.js';
+import { loggedErrorHandler } from './src/error/assert.js';
+
+// TODO possible additional exports. Some are privileged.
+// export { loggedErrorHandler };
+// export {
+//   makeCausalConsole,
+//   consoleLevelMethods,
+//   consoleOtherMethods,
+//   makeLoggingConsoleKit,
+//   filterConsole,
+//   pumpLogToConsole,
+// } from './src/error/console.js';
+// export { assertLogs, throwsAndLogs } from './src/error/throws-and-logs.js';
+
+/**
+ * `makeCausalConsoleFromLogger` is privileged because it exposes
+ * unredacted error info onto the `Logger` provided by the caller.
+ */
+export const makeCausalConsoleFromLogger =
```

Comment:
what does "causal" mean in this context? I can't find it explained in the repo. 

Also it would help for this export to explain why one would use it.

Reply:
Added explanations to ses/error/README.md, which already said a lot about it, but had not introduced the "causal console" terminology to name it.

Added a brief explanation here and a link to that README, where the relevant explanation will appear one this PR is merged.

Done. PTAL.